docs/storage/lumio/auth-lumidata-eu.md

```diff
@@ -131,3 +151,93 @@ can be created in its place.
   <figcaption>Access key details</figcaption>
 </figure>
 
+
+## Connecting
```

Comment:
I would not use term "connecting". Also I would keep this page for authorization only (tokens) and move "connecting" part to the next page "transferring and managing data".

Reply:
Since no further comments, marking this as resolved